CODE_OF_CONDUCT.md

```
@@ -2,127 +2,115 @@
+---
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org), version 2.0, available at:
 
-Community Impact Guidelines were inspired by [Mozilla's code of conduct
-enforcement ladder](https://github.com/mozilla/diversity).
+📄 https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)